### PR TITLE
Mobile, Drawer: Improve widget documentation readability

### PR DESCRIPTION
**Pull request by lslachciak**
[Problem]  Documentation contained mixed HTML and JS code in examples,
           Method return types in signatures were missing
[Solution] Code examples were separated and missing return types were added

Signed-off-by: Pawel Kaczmarczyk <p.kaczmarczy@samsung.com>

### CODE REVIEW

**Comment by huytaduc** [comment]
checkout the webpage https://docs.stg.tizen.org/staging/604

File: docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_Drawer.htm [hunk 1]
```diff
@@ -221,11 +221,13 @@ <h3>Summary</h3>
       &lt;li class=&quot;ui-drawer-sub-list&quot;&gt;&lt;a href=&quot;#&quot;&gt;Sub item 1&lt;/a&gt;&lt;/li&gt;
    &lt;/ul&gt;
 &lt;/div&gt;
-&lt;script&gt;
-   var drawerElement = document.getElementById(&quot;leftdrawer&quot;),
-       drawer = tau.widget.Drawer(drawerElement);
-   drawer.close();
-&lt;/script&gt;
+</pre>
+<p>JS code:</p>
```
Comment (mijin85cho) [inline docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_Drawer.htm]: Global Suggestion : How about adding "\<p>HTML code:\</p>" as "\<p>JS code:\</p>" added?

If you want to keep current state, I'll merge this PR.

**Comment by huytaduc** [comment]
checkout the webpage https://docs.stg.tizen.org/staging/604

**Comment by dorlinski** [comment]
fixed commit message

**Pull request merged**